src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work ⚙.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped its work.");

```

Comment:
The message printed should also be more informative about the type of machine. Instead of a generic message, specify that this is the Excavator that has stopped working.